docker-cpp-demos/amazonlinux-docker/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Build docker with
+# docker build -t kinesis-video-producer-sdk-cpp-amazon-linux .
+#
+FROM amazonlinux:latest
+RUN yum update -y && \
+	yum upgrade -y && \
+	yum install -y autoconf && \
```

Comment:
I think the packages can be installed with 1 yum install. E.g 

```
yum install -y \
  autoconf && \
  automake && \
  ...
```

Reply:
Done